### ISSUE TITLE
no style attribute in code blocks

### ISSUE THREAD

**Issue by pixelass** [revolunet/sublimetext-markdown-preview#346]
while this code works here on github I cannot use it with the plugin 
The code is wrapped in <code>```html </code>

``` html
<div class="Poti">
  <label>Volume</label>
  <div class="knob-wrapper" style="position:relative;height:100px;width:100px;">
    <div class="knob" style="position:absolute;transform:translate(-50%,-50%) rotate(30deg)"></div>
    <div class="markers" style="position:absolute;top:50%;left:50%">
      <div class="marker" style="position:absolute;top:0;left:0;transform:rotate(30deg) translateY(50px)">-10</div>
      <div class="marker" style="position:absolute;top:0;left:0;transform:rotate(150deg) translateY(50px)">0</div>
      <div class="marker" style="position:absolute;top:0;left:0;transform:rotate(330deg) translateY(50px)">10</div>
    </div>
  </div>
</div>
```

<img width="453" alt="screen shot 2016-07-04 at 1 19 40 pm" src="https://cloud.githubusercontent.com/assets/1148334/16558989/eaae25d8-41e9-11e6-915e-55c9c8d9fc2f.png">
<img width="384" alt="screen shot 2016-07-04 at 1 18 36 pm" src="https://cloud.githubusercontent.com/assets/1148334/16558990/eab10410-41e9-11e6-85ac-f501e63ab559.png">

**Comment by pixelass** [comment]
strangely I now get this on empty files too once it has been saved with this error.

I have to close and reopen sublime to get rid of the issue even if the file is empty.

**Comment by facelessuser** [comment]
Is it possible you exceeded your limit of calls to the Github API?  Maybe you need to provide authentication to remove that limit?  https://github.com/revolunet/sublimetext-markdown-preview#parsing-github-flavored-markdown-

**Comment by pixelass** [comment]
It seems the issue is not related to the `style` attribute. it is rather a warning which is not related to the actual error. It must have been a coincidence that it happened on the save when I added this block.

Thx for the suggestion about adding a token. It totally makes sense that I reached the limit, since I am writing a rather long article. I did not know the option with the auth-token existed. I'll try that and am pretty sure this will fix the issue.

The best..
